app/controllers/plants_controller.rb

```diff
@@ -29,6 +29,8 @@ def create
   end
 
   def category_hovercard
+    @category = Plant.friendly.find(params[:plant_id])&.category
+    
```

Comment:
In this part, the hovercard will include the category of the current plant in the text (see image)
<img width="345" alt="Screen Shot 2022-08-08 at 12 45 11 PM" src="https://user-images.githubusercontent.com/32690770/183410831-22bcaf36-b58d-4039-bd05-6a4a5451fa0a.png">